authorization/invitation/model/invitation_model_service.go

```diff
@@ -0,0 +1,166 @@
+package model
+
+import (
+	"context"
+	"fmt"
+	"github.com/fabric8-services/fabric8-auth/account"
+	"github.com/fabric8-services/fabric8-auth/authorization"
+	"github.com/fabric8-services/fabric8-auth/authorization/invitation"
+	invRepo "github.com/fabric8-services/fabric8-auth/authorization/invitation/repository"
+	permissionModel "github.com/fabric8-services/fabric8-auth/authorization/permission/model"
+	"github.com/fabric8-services/fabric8-auth/authorization/repository"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+)
+
+type InvitationModelService interface {
+	CreateInvitations(ctx context.Context, issuingUserId uuid.UUID, inviteTo uuid.UUID, invitations []invitation.Invitation) error
```

Comment:
Could just use `Create` and `List` ? to avoid stutter :) https://blog.golang.org/package-names

Reply:
Agreed, however I've renamed these methods differently now to account for different workflows.